src/components/Vega/EntropyPlot.js

```diff
@@ -29,8 +29,45 @@ const PlotContainer = styled.div``;
 
 const EntropyPlot = observer(({ width }) => {
   const vegaRef = useRef();
-  const { configStore, dataStore, UIStore, plotSettingsStore, snpDataStore } =
-    useStores();
+  const {
+    configStore,
+    dataStore,
+    UIStore,
+    plotSettingsStore,
+    snpDataStore,
+  } = useStores();
+
+  const nameToAbbr = {
+    "5'UTR": '5',
+    ORF1ab: 'ORF1ab',
+    ORF1a: 'ORF1a',
+    S: 'S',
+    ORF3a: '3a',
+    orf3a: '3a',
```

Comment:
We can't remove the "ORF" or "nsp" prefix from these names, they're integral to the name

Reply:
Same goes for all the other abbreviations you've made below. But you can definitely remove shorten, e.g., "nsp3 - PL2-PRO" -> "nsp3"

Reply:
Updated abbr to include ORF and nsp. I'm going to write logic to determine whether/where to place the label based on the size of the domain bar and the location of other nearby bars. In the meantime, users will have to scroll over the bars and check the tooltip to see the domain name.